lab_2_tokenize_by_bpe/main.py

```diff
@@ -47,6 +80,23 @@ def merge_tokens(
     :param pair: a pair of tokens to be merged
     :return: dictionary in the form of <preprocessed word: number of occurrences>
     """
+    if not (isinstance(word_frequencies, dict)
+            and isinstance(pair, tuple)):
+        return None
+    merged_frequencies = {}
+    for preprocessed_word, count in word_frequencies.items():
+        if ''.join(pair) in ''.join(preprocessed_word):
```

Comment:
looks like the you can remove `if` and else at all.

Reply:
I can only remove else